src/pages/use-cases/onboarding.tsx

```diff
@@ -179,22 +179,23 @@ const UseCasePage: FunctionComponent = () => (
                                     Decrease time to first commit for new developers, help existing engineers master
                                     your codebase, and fast-track full codebase understanding.
                                 </div>
-                                <div className="d-flex flex-column flex-lg-row pt-1">
-                                    <Link href="/demo" passHref={true}>
-                                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                                        <a
-                                            className="btn btn-primary mr-lg-3 mb-lg-0 mb-3 w-md-100"
-                                            title="Request a Demo."
-                                        >
-                                            Request a demo
-                                        </a>
-                                    </Link>
-                                    <Link href="/get-started" passHref={true}>
-                                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                                        <a className="btn btn-outline-primary w-md-100" title="Try Sourcegraph.">
-                                            Try Sourcegraph now
-                                        </a>
-                                    </Link>
+                                <div className="max-w-400 flex-column flex-md-row d-md-flex align-items-center">
+                                    <div className="mb-3 mb-md-0">
+                                        <Link href="/demo" passHref={true}>
+                                            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                                            <a className="btn btn-primary w-100" title="Request a Demo.">
```

Comment:
Looks like we're missing some tracking data attributes here

Reply:
I added them to the `a` tag on 2355098